### DISCUSSION TITLE
Trying to ignore some css line with a custom Hubspot selector

### DISCUSSION THREAD

**Discussion by jeremybdk** [webpack/webpack#16027] [Q&A]
Hello,

I'm trying to deploy an Hubspot theme built with Tailwind, I've decided to add Webpack + PostCss to allow for auto build of TailwindCSS + Import + SCSS support.

However I'm stuck with an issue:

Hubspot requires us to use the get_asset_url selector in css when you need to access an asset for background image for example.

So my CSS file (before process) will look like this:

`  .page-header-text-title {
    background-image: url("{{ get_asset_url('../images/page-name-doodle.svg') }}");
  }
`

Obviously this will crash as MiniCssExtractPlugin cannot process this line.

`ERROR in ./src/css/styles.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
HookWebpackError: Cannot find module '{{ get_asset_url('../images/page-name-doodle.svg') }}'`

I'm trying to figure what can I do to fix this, and maybe ignore this line when processing the file or creating a custom rule to ignore this.

Does someone know how to do this?

I've been trying for a few hours to find a good solution but cannot figure this out.

Thanks !

**Comment by alexander-akait** [comment]
You need to special loader to handle is, `url("{{ get_asset_url('../images/page-name-doodle.svg') }}");` is invalid CSS syntax, you can try to use the `url.filter` option for `css-loader` to ignore some kind of URLs

**Reply by jeremybdk** [reply]
Oh thank you so much, how would I do that?

**Reply by alexander-akait** [reply]
Here is an example https://github.com/webpack-contrib/css-loader#object

**Reply by jeremybdk** [reply]
Oh got it thank you so much !!
